packages/web3/src/transaction/utils.test.ts

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { getSigners } from '@alephium/web3-test'
+import { groupIndexOfTransaction } from './utils'
+import { ONE_ALPH } from '../constants'
+import { bs58, hexToBinUnsafe } from '../utils'
+import { unsignedTxCodec } from '../codec'
+import { groupOfAddress } from '../address'
+import { blake2b } from 'blakejs'
+import { web3 } from '../'
+import { PrivateKeyWallet } from '@alephium/web3-wallet'
+
+describe('transaction utils', () => {
+  beforeAll(() => {
+    web3.setCurrentNodeProvider('http://127.0.0.1:22973', undefined, fetch)
+  })
+
+  it('should calculate the group of transfer transaction', async () => {
+    const [signer1, signer2] = await getSigners(2)
```

Comment:
The `getSigners` function returns group 0 accounts by default. Maybe we can test the case where `fromGroup` and `toGroup` are not the same.